gym_electric_motor/physical_systems/mechanical_loads.py

```diff
@@ -11,10 +11,10 @@ class MechanicalLoad:
     It contains an mechanical ode system as well as the state names, limits and nominal values
     of the mechanical quantities. The only required state is 'omega' as the rotational speed of the motor shaft
     in rad/s.
-    ExternalSpeedLoad and ConstantSpeedLoad should be initialized with the own
-    initial value in class initialisation, when definded external
+    ExternalSpeedLoad and ConstantSpeedLoad can be initialized by giving the
+    initializer as an class parameter by instanziation

```

Comment:
As stated in the further annotations, externalSpeedLoad should be changed such that it ignores the initializer. 
(also please correct typo instanziation -> instantiation)